module-apps/apps-common/locks/handlers/SimLockHandler.cpp

```diff
@@ -113,7 +113,7 @@ namespace locks
         lock.attemptsLeft = attempts;
         lock.lockName     = utils::enumToString(Store::GSM::get()->selected);
 
-        if (simUnlockBlockOnLockedPhone) {
```

Comment:
you have a point there - it's much more readable right now, but I'm concerned a little bit with the underscore - I assume that you wanted to highlight that's an internal variable, but I don't know if we use this convention anywhere else?

Reply:
No, we don't, but we can start ;) This convention is probably better `mSomething` and surely better than putting leading underscore(s) because they can introduce an UB in some cases. But, I won't insist if a majority of the reviewers don't like it.

Reply:
sure - I'm not stubborn on this particular solution - but we should find one and the only one solution and write it down in a guideline ;)

Reply:
OK, will do in the future :) So, for now, I'll let myself leave this change as it is here